Handle invalid requirement as parameter to uninstall (#9052)
pypa/pip: news/4958.feature.rst

> Implemented error handling when passing an invalid requirement ``pip uninstall``.

Reviewer: ```suggestion
Add a warning when passing an invalid requirement to ``pip uninstall``.
```
